os/kernel/binary_manager/binary_manager.h

```diff
@@ -49,7 +49,7 @@
 /* Binary Manager Core Thread information */
 #define BINARY_MANAGER_NAME        "binary_manager"           /* Binary manager thread name */
 #define BINARY_MANAGER_STACKSIZE   2048                       /* Binary manager thread stack size */
-#define BINARY_MANAGER_PRIORITY    201                        /* Binary manager thread priority */
+#define BINARY_MANAGER_PRIORITY    203                        /* Binary manager thread priority */
```

Comment:
Could you add this in the title as well?

Reply:
yes